enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -49,20 +52,35 @@ export const InteractiveQuestionResult = ({
     onSave,
     isSaveEnabled,
     saveToCollectionId,
+    isCardIdError,
   } = useInteractiveQuestionContext();
 
   const [isSaveModalOpen, { open: openSaveModal, close: closeSaveModal }] =
     useDisclosure(false);
 
   // When visualizing a question for the first time, there is no query result yet.
-  const isQueryResultLoading = question && !queryResults;
+  const isQueryResultLoading =
+    question && shouldRunCardQuery(question) && !queryResults;
 
   if (isQuestionLoading || isQueryResultLoading) {
     return <SdkLoader />;
   }
 
-  if (!question) {
-    return <SdkError message={t`Question not found`} />;
+  // `isCardError: true` when the entity ID couldn't be resolved
+  if (!question || isCardIdError) {
+    return (
+      <SdkError
+        message={jt`Question ${(
+          <Code
+            bg="var(--mb-base-color-ocean-20)"
```

Comment:
Should we add a semantic variable for this?

Reply:
Agree, we shouldn't even use semantic colors directly like this. I just slapped something that I think it would make the ID stand out more.

@AlessioLaiso I think we need some design for the error component with ID.

Here's how it looks currently.
![image](https://github.com/user-attachments/assets/bc72c924-f79b-44b3-b19d-0fc63e3fc1fc)

I've added a todo list for cleaning up in [the description](https://github.com/metabase/metabase/pull/49714#issue-2641539254), thanks for reminding me this 🙏